lib/bolt/util.rb

```diff
@@ -85,6 +85,38 @@ def first_run?
         Bolt::Config.user_path && !File.exist?(first_runs_free)
       end
 
+      def find_file_in_mod(mod, module_file)
+        subdir = module_file.split(File::SEPARATOR).first
```

Comment:
You can use something like `subdir, relative_path = module_file.split(File::SEPARATOR, 2)` to split the path in only one place.